py_zipkin/encoding/protobuf/zipkin.proto

```diff
@@ -0,0 +1,214 @@
+// Copyright 2018 The OpenZipkin Authors
```

Comment:
maybe cite where this file is from, for maintenance pov?

Reply:
The makefile just pulls it from zipkin-api so I'd rather not modify it at all: https://github.com/drolando/py_zipkin/blob/add_protobuf_encoding/py_zipkin/encoding/protobuf/Makefile#L2